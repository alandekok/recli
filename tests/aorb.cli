a
b
hello
hello a
hello b
c hello
d hello
c
d

