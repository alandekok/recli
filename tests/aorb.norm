((c|d) hello|[hello] (a|b))
