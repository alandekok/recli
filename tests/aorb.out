a
b
hello
^ - Invalid input
hello a
hello b
c hello
d hello
c
^ - Invalid input
d
^ - Invalid input
